modules/server/main.tf

```diff
@@ -0,0 +1,374 @@
+  role    = "${local.server_project_roles[count.index]}"
+  project = "${var.project_id}"
+  member  = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_read" {
+  count  = "${var.org_id != "" ? length(local.server_read_roles) : 0}"
+  role   = "${local.server_read_roles[count.index]}"
+  org_id = "${var.org_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_folder_iam_member" "folder_read" {
+  count  = "${var.folder_id != "" ? length(local.server_read_roles) : 0}"
+  role   = "${local.server_read_roles[count.index]}"
+  folder = "${var.folder_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_write" {
+  count  = "${var.org_id != "" && var.enable_write ? length(local.server_write_roles) : 0}"
+  role   = "${local.server_write_roles[count.index]}"
+  org_id = "${var.org_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_folder_iam_member" "folder_write" {
+  count  = "${var.folder_id != "" && var.enable_write ? length(local.server_write_roles) : 0}"
+  role   = "${local.server_write_roles[count.index]}"
+  folder = "${var.folder_id}"
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+#------------------------#
+# Forseti Firewall Rules #
+#------------------------#
+resource "google_compute_firewall" "forseti-server-deny-all" {
+  name                    = "forseti-server-deny-all-${local.random_hash}"
+  project                 = "${local.network_project}"
+  network                 = "${var.network}"
+  target_service_accounts = ["${google_service_account.forseti_server.email}"]
+  source_ranges           = ["0.0.0.0/0"]
+  priority                = "200"
+
+  deny {
+    protocol = "icmp"
+  }
+
+  deny {
+    protocol = "udp"
+  }
+
+  deny {
+    protocol = "tcp"
+  }
+
+  depends_on = ["null_resource.services-dependency"]
+}
+
+resource "google_compute_firewall" "forseti-server-ssh-external" {
+  name                    = "forseti-server-ssh-external-${local.random_hash}"
+  project                 = "${local.network_project}"
+  network                 = "${var.network}"
+  target_service_accounts = ["${google_service_account.forseti_server.email}"]
+  source_ranges           = ["0.0.0.0/0"]
+  priority                = "100"
+
+  allow {
+    protocol = "tcp"
+    ports    = ["22"]
+  }
+
+  depends_on = ["null_resource.services-dependency"]
+}
+
+resource "google_compute_firewall" "forseti-server-allow-grpc" {
+  name                    = "forseti-server-allow-grpc-${local.random_hash}"
+  project                 = "${local.network_project}"
+  network                 = "${var.network}"
+  target_service_accounts = ["${google_service_account.forseti_server.email}"]
+  source_ranges           = ["10.128.0.0/9"]
```

Comment:
I think that this should be parametrizable. Forseti client can be on 10.0.0.0/24 subnetwork for example.

Reply:
Good point, I've pushed this to an issue that we can address independently.